.gitignore

```diff
@@ -6,3 +6,11 @@
 cover.out
 /vendor
 /.vendor-new
+
+# gitlab example
+cluster/examples/gitlab/gcp/values-psql.yaml
```

Comment:
nit: creating a local `.gitignore` file in `cluster/examples/gitlab` would put these rules closer to the code using them.

Reply:
:heavy_check_mark: